acceptancetests/reporting.py

```diff
@@ -0,0 +1,90 @@
+"""Reporting helper class for communicating with influx db."""
+import abc
+import time
+import datetime
+try:
+    import urlparse
+except ImportError:
+    import urllib.parse as urlparse
+
+from influxdb import InfluxDBClient
+from influxdb.client import InfluxDBClientError
+
+__metaclass__ = type
+
+DBNAME = 'txn_metrics'
+POLICYNAME = 'txn_metric'
+
+class _Reporting:
+    """_Reporting represents a class to report metrics upon"""
+
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(self, client):
+        self.client = client
+        self.labels = {
+            "total_time": "txn_metric.total_time",
+            "total_num_txns": "txn_metric.total_num_txns",
+            "max_time": "txn_metric.max_time",
+            "test_duration": "txn_metric.test_duration",
+        }
+
+    @abc.abstractmethod
+    def report(self, metrics, tags):
+        """Report the metrics to the underlying reporting client
+        """
+
+class InfluxDB(_Reporting):
```

Comment:
If this is the reporting client let's call it something like that so it doesn't appear to be something from the influxdb package. 

Maybe just InfluxClient or even ReportingClient?